### ISSUE TITLE
[feature]: Remove requirement for last name

### ISSUE THREAD

**Issue by devSparkle** [makeplane/plane#2511]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Eliminate the requirement to input a last name when setting up an account, or updating a user profile. While this can already be bypassed by inputting a space; formalising support would be better.

### Why should this be worked on?

This request is based on two use cases:

1. Many people do not have a last name! Forcing them to input one can be awkward for anyone in this particular group.
2. Some teams, especially remote-only may refer to each-other through nicknames. Forcing them to input a last name can make it harder to know who is who.

**Comment by aaryan610** [comment]
Hi @devSparkle, thanks for raising the concern. Our team will think on it and come up with a good solution.

**Comment by theparthacus** [comment]
good point and seems silly to have it if it can be bypassed with a period or a space. we took inspiration from Notion and Slack with the rationale that teams that need to have a last name can have it. we will make it optional.

**Comment by sebzz2k2** [comment]
@vihar @sriramveeraghanta can i work on this issue ?

**Comment by vihar** [comment]
Last name is still required — the field has a required rule, so entering a space is still the workaround.

Not on our roadmap. Closing this out.
